packages/starlight/utils/user-config.ts

```diff
@@ -231,6 +231,15 @@ const UserConfigSchema = z.object({
    * })
    */
   customCss: z.string().array().optional().default([]),
+
+  /** Define if the last update date should be visible in the page footer. */
+  showUpdateDate: z
+    .boolean()
+    .optional()
+    .default(false)
+    .describe(
+      'Define if the last update date should be visible in the page footer.'
+    ),
```

Comment:
We can skip the `optional()` call here as it is implied by `default()`:

```suggestion
  lastUpdated: z
    .boolean()
    .default(false)
    .describe(
      'Define if the last update date should be visible in the page footer.'
    ),
```